style.css

```diff
@@ -0,0 +1,51 @@
+body {
+    background-color: black;
+  }
+  
+  nav {
+    display: flex;
```

Comment:
Flex model used correctly, keep it going!

Reply:
- [ ] Kindly check my comments on line `29` above 👍